app/models/serial_work.rb

```diff
@@ -21,4 +23,15 @@ def delete_empty_series
       self.series.destroy
     end
   end
+
+  # Ensure series bookmarks are reindexed when a new work is added to a series
+  def update_series_index
+    if $rollout.active?(:start_new_indexing)
+      Series.all.each(&:enqueue_to_index)
```

Comment:
Sorry, I think I gave you the wrong idea here with my phrasing -- it's not necessary to reindex all *series*, but it is necessary to reindex all *bookmarks* for the series that was modified, just as in the old indexing code below.